### PR TITLE
Add executor binary crate

### PR DESCRIPTION
**Pull request by danielSanchezQ**

### CODE REVIEW

**Comment by danielSanchezQ** [comment]
@bacv, we can merge this before or after the DA changes what you think is gonna be easier.

**Comment by bacv** [comment]
> @bacv, we can merge this before or after the DA changes what you think is gonna be easier.

Executor won't be usable in tests until it has the ability to encode and disperse blobs. I'm fine with holding #764 unmerged until we have this functionality.

**Comment by danielSanchezQ** [comment]
> > @bacv, we can merge this before or after the DA changes what you think is gonna be easier.
> 
> Executor won't be usable in tests until it has the ability to encode and disperse blobs. I'm fine with holding #764 unmerged until we have this functionality.

No, #764 should go ahead, its working so its fine. Same tests will have to pass when we have the executor. It will imply changing the tests a bit, but not so much. 

Lets merge #764 and then ill rebase this if needed.

**Comment by danielSanchezQ** [comment]
Unrelated error.

File: nodes/nomos-executor/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                registry: registry.clone(),
+            },
+            da_network: config.da_network,
+            da_indexer: config.da_indexer,
+            da_sampling: config.da_sampling,
+            da_verifier: config.da_verifier,
+            cryptarchia: config.cryptarchia,
+            #[cfg(feature = "metrics")]
+            metrics: MetricsSettings { registry },
+            storage: RocksBackendSettings {
```
Comment (bacv) [inline nodes/nomos-executor/src/main.rs:90]: Nit: recently this was made configurable via the config, we can remove the const and just use config.storage

**Review by bacv — APPROVED** [review]

File: nodes/nomos-executor/src/main.rs (hunk 1 above)
Comment (danielSanchezQ) [reply nodes/nomos-executor/src/main.rs:90]: Nice!, will add, thanks!

**Pull request merged**